tests/move/test_move_core.py

```diff
@@ -510,3 +514,114 @@ def test_move_core(self, tmp_config):
         tmp_config(settings='default_plugins = ["move"]')
 
         assert config.CONFIG.pm.has_plugin("move_core")
+
+
+class TestPluginOverrideAlbumPathConfig:
+    """Test plugin that implements the override_album_path_config hook."""
+
+    @staticmethod
+    @moe.hookimpl
+    def override_album_path_config(album: Album) -> Optional[str]:
+        """Override the `album_path` for classical music albums and soundtracks."""
+        if "Classical" in album.title:
+            return "Classical/{album.artist}/{album.title} ({album.year})"
+        elif "Soundtrack" in album.title:
+            return "Soundtracks/{album.title} ({album.year})"
+        return None
```

Comment:
Not sure if you mean to explicitly return None for clarity purposes, but it's not necessary in Python.